pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -643,7 +643,7 @@
   highlights_constant_athletes_short_heading: Push yourself
 
   footer_copyright_message: 'Copyright %{current_year} Code.org. All rights reserved.'
-  footer_trademark_message: 'Hour of Code™ is a trademark of Code.org.'
+  footer_trademark_message: 'Hour of Code™ and Hora del Código® are trademarks of Code.org.'
```

Comment:
Are we worried about old translations not including Hora del Código®?

Reply:
No, it's not urgent. Jorge and I will follow up in Crowdin